CMakeLists.txt

```diff
@@ -746,7 +757,9 @@ endif()
 
 # Should we cross-compile the standard library for Android?
 is_sdk_requested(ANDROID swift_build_android)
-if(swift_build_android AND NOT "${SWIFT_ANDROID_NDK_PATH}" STREQUAL "")
+if(swift_build_android
+    AND NOT "${SWIFT_ANDROID_NDK_PATH}" STREQUAL ""
```

Comment:
I think it would be nicer if we sink the `if (NOT ${SWIFT_ANDROID_NDK_PATH} STREQUAL "")` into the case, as an always thing.

```
if(swift_build_android AND NOT ${SWIFT_HOST_VARIANT_SDK} STREQUAL ANDROID)
  if("${SWIFT_ANDROID_NDK_PATH}" STREQUAL "")
    message(FATAL_ERROR "SWIFT_ANDROID_NDK_PATH must be set to the Android NDK path")
  endif()
```